ui/pages/home/home.container.js

```diff
@@ -223,8 +222,10 @@ const mapStateToProps = (state) => {
     custodianDeepLink: getCustodianDeepLink(state),
     accountType: getAccountType(state),
     ///: END:ONLY_INCLUDE_IF
-    isSmartTransactionsOptInModalAvailable:
-      getIsSmartTransactionsOptInModalAvailable(state),
+
+    // Set to false to prevent the opt-in modal from showing.
+    // TODO(dbrans): Remove opt-in modal once default opt-in is stable.
```

Comment:
If the modal can never show in this build, then why not remove it in this PR also if it's unused code?

Reply:
`DEFAULT_SMART_TRANSACTIONS_ENABLED` is being used as a feature flag. If we need to disable smart transactions by default we can just flip that boolean and revert the home container.

Reply:
Is it a literal remote feature flag or do you just mean it's easy to enable them again?

I appreciate that, but could we also just revert a commit to bring all the modal code back if needed?

My main point is that in this commit the modals cannot be shown so it's dead code.

Reply:
> or do you just mean it's easy to enable them again

Right it's just easy to re-enable.

Matt, how do you feel about cleaning up the dead code (selectors, components, etc.) in a separate PR? I think it might make things simpler for this one, letting this PR focus just on changing the default preferences.